projects/lights_control/test/test_lights_blinker.c

```diff
@@ -0,0 +1,130 @@
+#include <stdlib.h>
+
+#include "interrupt.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#include "lights_blinker.h"
+
+void setup_test(void) {
+  interrupt_init();
+  soft_timer_init();
+  event_queue_init();
+}
+
+void teardown_test(void) {}
+
+void test_lights_blinker_init(void) {
+  LightsBlinker blinker;
+  lights_blinker_init(&blinker);
+  TEST_ASSERT_EQUAL(blinker.timer_id, SOFT_TIMER_INVALID_TIMER);
+}
+
+void test_lights_blinker_on(void) {
+  LightsBlinker blinker_us;
+  LightsBlinker blinker_millis;
+  LightsBlinkerDuration duration_us = 300000;
+  LightsBlinkerDuration duration_millis = 500;
+
+  EventID EVENT_1 = 1;
+  EventID EVENT_2 = 2;
+
+  TEST_ASSERT_OK(lights_blinker_init(&blinker_us));
+  TEST_ASSERT_OK(lights_blinker_init(&blinker_millis));
+
+  TEST_ASSERT_OK(lights_blinker_on_us(&blinker_us, duration_us, EVENT_1));
```

Comment:
Can you add comments explaining the intention behind this test? It's a little confusing as to what each block is supposed to be doing.